Symfony/CS/Tests/Console/ConfigurationResolverTest.php

```diff
@@ -566,4 +566,28 @@ public function testResolveUsingCacheWithNegativeConfigAndNoOption()
 
         $this->assertFalse($this->config->usingCache());
     }
+
+    public function testResolveCacheFileWithoutOption()
```

Comment:
need to test complete matrix like all other options.
- [ ] set none
- [x] set only in config
- [x] set only as CLI option
- [ ] set both